CHANGELOG.md

```diff
@@ -114,6 +122,19 @@ These should not affect the functionality of the module.
 
 - dns\_lookup.rb: support arrays of names [\#1](https://github.com/voxpupuli/puppet-dnsquery/pull/1) ([pcarrier](https://github.com/pcarrier))
 
+3.0.0:
```

Comment:
those entries appear after the 0.1.0 entries, that does not make any sense

Reply:
ahh yes i removed them manually last time, they are being pulled in by the rack task but i have not looked into why.  i have removed manually again for now